modular_skyrat/modules/modular_disabler/beams.dm

```diff
@@ -0,0 +1,84 @@
+/obj/projectile/beam/disabler/weak
+	name = "weakened disabler beam"
+	damage = 30 // TG default
+
+/obj/projectile/beam/disabler/weak/split
+	damage = 12
+
+/obj/projectile/beam/disabler/weak/untie
+	damage = 15
+	light_color = COLOR_SAMPLE_BROWN
+
+/obj/projectile/beam/disabler/weak/untie/on_hit(atom/target, blocked, pierce_hit)
+	. = ..()
+	var/mob/living/carbon/hit = target
```

Comment:
you could typecast the `target` argument to mob/living/carbon and just do an early return if an `istype(target)` fails